.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Continuous Integration
+concurrency: 
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [ master ]
```

Comment:
So the idea is to have users verifying PRs on their own quota and not ours :stuck_out_tongue_winking_eye: Do we need the `pull_request` at all ? Are users really creating pull request against their own fork ? I would think the most common workflow is just to push on any branch in their repo.

This would be duplicating what's done on https://ci.eclipse.org/jdt/job/eclipse.jdt.core-Github/job/master/ though right ?

Reply:
> Do we need the pull_request at all ?

Indeed, I don't think we need it. I'll remove it.

> Are users really creating pull request against their own fork ?

If they want, they can change this file on their fork.

> This would be duplicating what's done on https://ci.eclipse.org/jdt/job/eclipse.jdt.core-Github/job/master/ though right ?

Yes, but I think it's a good way to ensure this build works; and I hope it wouldn't be annoying.

Reply:
If it does become annoying (or the verification ends up taking longer, or conflicting results), I think you might be able to use `$GITHUB_REPOSITORY_OWNER` (should be eclipse-jdt here) to distinguish between fork/non-fork. It'd have to be used after the job is already triggered so not the cleanest solution.